### PR TITLE
create Ray-Allen

### PR DESCRIPTION
**Pull request by dechengc**
Please review this, @casey-collab

### CODE REVIEW

File: Chinese/Ray-Allen/Ray-Allen.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Walter Ray Allen, Jr. (born July 20, 1975) is an American professional basketball player who is currently a free agent.
```
Comment (casey-collab) [inline Chinese/Ray-Allen/Ray-Allen.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by dechengc** [comment]
I have updated the existing story. Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :octocat:

**Pull request merged**